dex/elrond_dex_pair/src/lib.rs

```diff
@@ -528,41 +507,14 @@ pub trait Pair:
         amount: &Self::BigUint,
         destination: &Address,
         opt_accept_funds_func: &OptionalArg<BoxedBytes>,
-    ) -> SCResult<()> {
+    ) {
         if amount > &0 {
-            let (function, gas_limit) = match opt_accept_funds_func {
-                OptionalArg::Some(accept_funds_func) => (
-                    accept_funds_func.as_slice(),
-                    self.transfer_exec_gas_limit().get(),
-                ),
-                OptionalArg::None => {
-                    let no_func: &[u8] = &[];
-                    (no_func, 0u64)
-                }
-            };
-
-            let result = self.send().direct_esdt_execute(
-                destination,
-                token,
-                amount,
-                gas_limit,
-                function,
-                &ArgBuffer::new(),
-            );
-
-            match result {
-                Result::Ok(_) => Ok(()),
-                Result::Err(_) => {
-                    sc_error!("Direct esdt nft execute failed")
-                }
-            }
-        } else {
-            Ok(())
+            self.send_fft_tokens(token, amount, destination, opt_accept_funds_func);
```

Comment:
The `send_tokens` function just call `send_fft_tokens`, why not use `send_fft_tokens` directly?

Reply:
To avoid using the "if" multiple times

Reply:
I see. I missed that when reviewing the change.